src/test/java/com/google/sps/UtilityTest.java

```diff
@@ -16,12 +16,15 @@
 
 import com.google.sps.classes.SubjectTag;
 import com.google.sps.classes.Utility;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
```

Comment:
I guess there is no use for the response import.

Reply:
Removed HttpServletResponse import!